alerting-config-service-impl/src/main/java/org/hypertrace/alerting/config/service/EventConditionConfigServiceRequestValidator.java

```diff
@@ -0,0 +1,46 @@
+package org.hypertrace.alerting.config.service;
+
+import com.google.common.base.Preconditions;
+import org.hypertrace.alerting.config.service.v1.CreateEventConditionRequest;
+import org.hypertrace.alerting.config.service.v1.DeleteEventConditionRequest;
+import org.hypertrace.alerting.config.service.v1.GetAllEventConditionsRequest;
+import org.hypertrace.alerting.config.service.v1.UpdateEventConditionRequest;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+
+public class EventConditionConfigServiceRequestValidator {
+
+  public void validateCreateEventConditionRequest(
```

Comment:
See validator PR for new utils that may make this a bit easier and more uniform (things like preconditions will be wrapped as an internal error in GRPC).

Reply:
done